File: c/soma_impares.c
#include <stdio.h>

int main(){
    int x, y, soma, troca;

    printf("Digite dois numeros:\n");
    scanf("%d %d", &x, &y);

    if (x > y) {
        troca = x;
        x = y;
        y = troca;
    }

	soma = 0;
    for (int i=x+1; i<y; i++) {
        if (i % 2 != 0) {
            soma = soma + i;
        }
    }

    printf("SOMA DOS IMPARES = %d\n", soma);

    return 0;
}

File: python/soma_impares.py
x: int; y: int; soma: int; troca: int;

print("Digite dois numeros:")
x = int(input())
y = int(input())

if x > y:
	troca = x
	x = y
	y = troca

soma = 0;
for i in range(x+1, y):
	if i % 2 != 0:
		soma = soma + i

print(f"SOMA DOS IMPARES = {soma}")